deploy/template/tutorial-web-app.yml

```diff
@@ -68,7 +68,7 @@ objects:
             value: ${SSO_ROUTE}
           - name: WALKTHROUGH_LOCATIONS
             value: ${WALKTHROUGH_LOCATIONS}
-          image: quay.io/integreatly/tutorial-web-app:v2.7.0
+          image: quay.io/integreatly/tutorial-web-app:v2.8.0
```

Comment:
this tag is still missing. I think you can just re-tag `latest`

Reply:
this exists now